crates/re_renderer/src/resource_managers/texture_manager.rs

```diff
@@ -16,20 +16,38 @@ use crate::{
 pub struct GpuTexture2DHandle(GpuTexture);
 
 impl GpuTexture2DHandle {
+    #[inline]
+    pub fn texture(&self) -> &GpuTexture {
+        &self.0
+    }
+
+    #[inline]
+    pub fn wgpu_handle(&self) -> crate::wgpu_resources::GpuTextureHandle {
+        self.0.handle
+    }
```

Comment:
Let's call this just `handle` since what's returned isn't an actual wgpu object in any way. Seeing this name I'd expect a `wgpu::Something` type.

In the same vain, we should rename `GpuTexture2DHandle` to `GpuTexture2D`, up to you if you want to do that in the same PR or another one